src/client/src/mockData/azureServiceOptions.ts

```diff
@@ -59,6 +59,14 @@ export const azureMessages = defineMessages({
     defaultMessage:
       "Connect your web app to a distributed database service to access and query data using SQL or MongoDB API."
   },
+  cosmosTimeExpectation: {
+    id: "cosmosDb.timeExpectation",
+    defaultMessage: "12 months of free access"
```

Comment:
It's not 12 months of free access forever though right (see later sentence for explanation of what I mean)? And the free access has limits right? Ignoring everything I know about Azure. I think this message conveys that I can use as much of cosmos  possibly even 10 Gb-1 Tb of storage for free for 12 months, which I don't think is true. Also may convey that every time the user creates a new instance they get 12 month's free which I'm not sure is true either. Correct me if i'm wrong here. I think you should have some more clear wording for these things or an annotation saying that it's only free from the first instance's launch up to a certain point, also linking the hard limits per month if any.